src/node_jsvmapi.cc

```diff
@@ -499,7 +499,7 @@ namespace v8impl {
     CHECK_TO_TYPE(Number, context, result, src, napi_string_expected)
```

Comment:
Apologies I didn't add a separate comment earlier, but I believe this one is also a copy/paste mistake.